nemo/collections/tts/torch/g2ps.py

```diff
@@ -388,7 +388,7 @@ def parse_one_word(self, word: str):
             return word, True
 
         # Punctuation (assumes other chars have been stripped)
-        if re.search("[a-zA-Z]", word) is None:
+        if re.search("[a-zA-ZÀ-ÿ0-9]", word) is None:
             return list(word), True
```

Comment:
Can you add a test for `class IPAG2P(BaseG2p)` that it will do what you expect against in-vocab and oov words?

Reply:
The tests for TTS in general right now seem very sparse. I will see what is appropriate to add in the context of this PR. IPAG2P might be a bit difficult to test, but the individual functions/modules it uses (eg. work tokenize, oov word functions, dictionary parsing, parse 1 word) should be doable.